qemu/tests/block_copy.py

```diff
@@ -282,9 +283,9 @@ def get_backingfile(self, method="monitor"):
         if method == "monitor":
             return self.vm.monitor.get_backingfile(self.device)
 
-        qemu_img = utils_misc.get_qemu_img_binary(self.params)
-        cmd = "%s info %s " % (qemu_img, self.get_image_file())
-        info = utils.system_output(cmd)
+        qemu_img = qemu_storage.QemuImg(self.params, self.data_dir, self.tag)
+        qemu_img.image_filename = self.get_image_file()
+        info = qemu_img.info()
```

Comment:
`force_share = True`  ?

Reply:
@yanan-fu Ah...yes, I have tested on qemu2.9, therefore passed, thanks for pointing out.